bootstrap/s3_bucket.tf

```diff
@@ -37,3 +49,54 @@ resource "aws_s3_bucket" "bucket" {
     "Account"     = "${var.aws_account_id}"
   }
 }
+
+resource "aws_s3_bucket_policy" "terraform_remote_state" {
+  bucket = "${aws_s3_bucket.bucket.id}"
+  policy = "${data.aws_iam_policy_document.terraform_remote_state.json}"
+}
+
+data "aws_iam_policy_document" "terraform_remote_state" {
+  statement {
+    sid       = "RequireEncryptedTransport"
+    effect    = "Deny"
+    actions   = ["s3:*"]
+    resources = ["${aws_s3_bucket.bucket.arn}/*"]
+    condition {
+      test     = "Bool"
+      variable = "aws:SecureTransport"
+      values   = ["false"]
+    }
+    principals {
+      type        = "*"
+      identifiers = ["*"]
+    }
+  }
+  
```

Comment:
And here

Reply:
Fixed.

Thanks for your feedback, Ed.  You're absolutely right - I've made those changes now.  Please let me know if these changes don't resolve the points you've raised.